src/org/omegat/Bundle.properties

```diff
@@ -1800,6 +1800,7 @@ WG_SAVE_ORIGIN=Save origin of &translation
 WG_INITIAL_SEGMENT_LOAD_COUNT=Initial number of loaded segments:
 WG_INITIAL_SEGMENT_LOAD_COUNT_TOOLTIP=When a single file has more segments than this, additional segments will be loaded on-demand as you scroll.
 WG_PARA_MARK=Paragraph delimiter:
+WF_OPTION_ALLOW_SELECT_SINGLE_CLICK=Activate segment with single click (Feature is only active while the cursor is locked.)
```

Comment:
- The layout in the preference window does not work. The string is too long and pushes the other half of the window all the way to the left.
- The comment in parenthesis is due in the manual, not in the preference.